migrations/2_deploy_contracts.js

```diff
@@ -16,17 +14,5 @@ module.exports = function(deployer, network, accounts) {
       "0x112234455c3a32fd11230c42e7bccd4a84e02010",
       dns.hexEncodeName("xyz."),
       namehash.hash("xyz"));
-  } else {
```

Comment:
Did you intend to remove this from the deploy script?

Reply:
I took it out because automated test does not require this (it creates within a test), but may put it back so that people can manually deploy on ganache. Do we need this only for `development` purpose and ignore for `mainnet` use now?